### DISCUSSION TITLE
UserWarning: Sparse CSR tensor support is in beta state

### DISCUSSION THREAD

**Discussion by antoni4040** [pyg-team/pytorch_geometric#7140] [Q&A] [answered]
After upgrading pytorch geometric, I started getting this:
`/home/antoni4040/anaconda3/envs/condanew/lib/python3.10/site-packages/torch_sparse/matmul.py:97: UserWarning: Sparse CSR tensor support is in beta state. If you miss a functionality in the sparse tensor support, please submit a feature request to https://github.com/pytorch/pytorch/issues. (Triggered internally at ../aten/src/ATen/SparseCsrTensorImpl.cpp:54.)`

I'm not sure how this is triggered, cause I don't use any sparse tensors from my side. Any ideas?

**Comment by EdisonLeeeee** [comment]
PyG 2.3 is attempting to replace `torch_sparse` with PyTorch native sparse tensors (e.g. `CSR`), and the warning is coming from PyTorch 2.0.

**Comment by rusty1s** [comment]
Yeah, this warning is a bit annoying, sorry about that. You can disable it via
```python
import warnings
warnings.filterwarnings('ignore', '.*Sparse CSR tensor support is in beta state.*')

**Comment by antoni4040** [comment]
I figured out that it must come from some positional encodings I'm using. 
Are there actual known problems with it or is it a warning-just-in-case?

**Reply by rusty1s** [reply, marked as answer]
There is no known problem, and everything works as expected.

**Comment by Phelixh** [comment]
Same happened. There is no known problem, and everything works as expected.
